package.json

```diff
@@ -38,13 +38,14 @@
         "classnames": "^2.3.2",
         "libphonenumber-js": "^1.10.44",
         "luxon": "^3.4.3",
+        "react": "^16.9.24",
+        "react-dom": "^16.9.24",
```

Comment:
Moved from `peerDeps` to `deps` to avoid conflict in parent project like in Storybook.